packages/plugin-expo-app/app.js

```diff
@@ -16,23 +16,29 @@ module.exports = {
       lastState = newState
     })
 
-    let nativeBundleVersion, nativeVersionCode
+    let nativeBundleVersion, nativeVersionCode, bundleVersion, versionCode
 
-    if (Constants.appOwnership === 'standalone') {
+    if (Constants.appOwnership !== 'expo') {
       if (Constants.platform.ios) {
         nativeBundleVersion = Application.nativeBuildVersion
+        bundleVersion = Application.nativeBuildVersion
       }
 
       if (Constants.platform.android) {
         nativeVersionCode = Application.nativeBuildVersion
+        versionCode = Application.nativeBuildVersion
       }
     }
 
     client.addOnSession(session => {
-      if (Constants.manifest?.revisionId) {
-        session.app.codeBundleId = Constants.manifest.revisionId
-      } else if (Constants.manifest2?.extra?.expoClient?.revisionId) {
-        session.app.codeBundleId = Constants.manifest2.extra.expoClient.revisionId
+      if (client._config.codeBundleId) {
+        session.app.codeBundleId = client._config.codeBundleId
+      }
+
+      if (versionCode) {
+        session.app.versionCode = versionCode
+      } else if (bundleVersion) {
+        session.app.bundleVersion = bundleVersion
```

Comment:
Do we need these `if`s? Presumably there's no default for these on `session.app`, so this would be equivalent:

```suggestion
      session.app.versionCode = versionCode
      session.app.bundleVersion = bundleVersion
```